sbin/common-functions.sh

```diff
@@ -21,8 +21,8 @@ FREETYPE_FONT_VERSION=${FREETYPE_FONT_VERSION:-2.4.0}
 
 determineBuildProperties() {
 
-    OS_KERNEL_NAME=$(uname | awk '{print tolower($0)}')
-    OS_MACHINE_NAME=$(uname -m)
+    export OS_KERNEL_NAME=$(uname | awk '{print tolower($0)}')
+    export OS_MACHINE_NAME=$(uname -m)
```

Comment:
Minor: `OS_MACHINE_NAME` is the value used in the global context, otherwise might not be necessary to make it a global var